GooglePlayInstant/Samples/SphereBlast/scripts/WoodPanel.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class WoodPanel : MonoBehaviour {
+    private AudioSource woodPanelClip;
+    private Hashtable played;
```

Comment:
Can probably use HashSet<string> for this case.

Reply:
no longer needed; removed